Assets/Modules/Enemies/ProjectileDestroyController.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+public class ProjectileDestroyController : MonoBehaviour
+{
+
+    private float timer;
+
```

Comment:
Also, ensure that your VS Code has a C# prettifier plugin and learn how to make the scripts pretty using the intended shortcut. The prettifiers will always remove unnecessary spaces and indents.

Reply:
ok I found one called CSharpier hope it's good even though I don't  love that they put SerializedField on top and then the variable on the bottom. Otherwise it helped! thanks!